apps/spruce/src/types/host.ts

```diff
@@ -67,6 +67,7 @@ export enum HostEvent {
   HostTeardown = "HOST_TEARDOWN",
   HostTaskFinished = "HOST_TASK_FINISHED",
   HostExpirationWarningSet = "HOST_EXPIRATION_WARNING_SENT",
```

Comment:
this is unrelated to your changes but might be a quick fix to update this one (it has a typo)
```suggestion
  HostExpirationWarningSent = "HOST_EXPIRATION_WARNING_SENT",
```